gazebo_ros/launch/range_world.launch

```diff
@@ -9,17 +9,20 @@
   <arg name="debug" default="false"/>
   <arg name="physics" default="ode"/>
   <arg name="verbose" default="true"/>
+  <arg name="output" default="screen"/>
   <arg name="world" default="gazebo_ros_range"/>
 
-  <!-- set use_sim_time flag -->
-  <group if="$(arg use_sim_time)">
-    <param name="/use_sim_time" value="true" />
-  </group>
-
   <include file="$(find gazebo_ros)/launch/empty_world.launch">
     <arg name="world_name" value="$(find gazebo_plugins)/test/test_worlds/$(arg world).world"/>
+    <arg name="paused" value="$(arg paused)"/>
+    <arg name="use_sim_time" value="$(arg use_sim_time)"/>
+    <arg name="extra_gazebo_args" value="$(arg extra_gazebo_args)"/>
     <arg name="gui" value="$(arg gui)"/>
     <arg name="debug" value="$(arg debug)"/>
+    <arg name="physics" value="$(arg physics)"/>
+    <arg name="verbose" value="$(arg verbose)"/>
+    <arg name="output" value="$(arg output)"/>
+    <arg name="world" value="$(arg world)"/>
```

Comment:
This is causing test failures:

~~~
14:38:36   /opt/ros/kinetic/share/rostest/cmake/../../../bin/rostest --pkgdir=/tmp/ws/src/gazebo_ros_pkgs/gazebo_plugins --package=gazebo_plugins --results-filename test_range_range_plugin.xml --results-base-dir "/tmp/ws/test_results" /tmp/ws/src/gazebo_ros_pkgs/gazebo_plugins/test/range/range_plugin.test 
14:38:36 ... logging to /home/buildfarm/.ros/log/rostest-8335a4ee6979-3476.log
14:38:36 Traceback (most recent call last):
14:38:36   File "/opt/ros/kinetic/share/rostest/cmake/../../../bin/rostest", line 36, in <module>
14:38:36     rostestmain()
14:38:36   File "/opt/ros/kinetic/lib/python2.7/dist-packages/rostest/__init__.py", line 274, in rostestmain
14:38:36     _main()
14:38:36   File "/opt/ros/kinetic/lib/python2.7/dist-packages/rostest/rostest_main.py", line 159, in rostestmain
14:38:36     testCase = rostest.runner.createUnitTest(pkg, test_file, options.reuse_master, options.clear, options.results_base_dir)
14:38:36   File "/opt/ros/kinetic/lib/python2.7/dist-packages/rostest/runner.py", line 229, in createUnitTest
14:38:36     config = roslaunch.parent.load_config_default([test_file], None)
14:38:36   File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/config.py", line 455, in load_config_default
14:38:36     raise RLException(e)
14:38:36 roslaunch.core.RLException: unused args [world] for include of [/tmp/ws/src/gazebo_ros_pkgs/gazebo_ros/launch/empty_world.launch]
~~~

Reply:
I removed the argument, lets see if CI succeeds